File: src/commands/paste.js
const paste = require("../utils/paste");

module.exports = {
    name: "paste",
    description: "Paste the message to dpaste.com and return the link.",
    usage: "<prefix>paste <message>",
    private: false,
    async execute(message, args) {
        const content = args.join(" ");

        if (!content) {
            return message.channel.send("You need to provide a message to paste.");
        }

        const url = await paste(content);

        if (!url) {
            return message.channel.send("There was an error while pasting the message.");
        }

        const authorId = message.author.id;
        message.channel.send(`<@${authorId}> your message has been pasted: ${url}`);
    }
}

File: src/commands/paste.rs
// paste.rs
use crate::util;
use serenity::model::channel::Message;
use serenity::prelude::Context;

pub async fn execute(ctx: Context, msg: &Message, args: Vec<&str>) -> Result<(), Box<dyn std::error::Error>> {
    let content = args.join(" ");
    let mut url = util::paste(&content).await?;
    let user_id = msg.author.id.to_string();
    url = format!("<@{}> has pasted:\n {}", user_id, url);
    msg.channel_id
        .say(&ctx.http, &url)
        .await?;

    Ok(())
}